### PR TITLE
Rename default storage driver from xen to file

### PR DESCRIPTION
**Pull request by kalkin**
- Rename XenPool   ⇒ FilePool
- Rename XenVolume ⇒ FileVolume

### CODE REVIEW

File: qubes/storage/__init__.py [hunk 1]
```diff
@@ -287,7 +287,7 @@ def stop(self, volume):
         raise NotImplementedError("Pool %s has stop() not implemented" %
                                   self.name)
 
-    def init_volume(self, volume_config):
+    def init_volume(self, vm, volume_config):
```
Comment (marmarek) [inline qubes/storage/__init__.py]: Is this change really related to renaming?

Comment (kalkin) [inline qubes/storage/__init__.py]: This line kind of slipped in, but is needed in general. Should i strip it out and do an own PR for that?

Comment (marmarek) [inline qubes/storage/__init__.py]: Maybe not own PR, but separate commit.

**Comment by kalkin** [comment]
@marmarek I splitted up the commit
